piccolo_api/fastapi/endpoints.py

```diff
@@ -120,6 +120,17 @@ def _get_type(type_: t.Type) -> t.Type:
     return type_
 
 
+def _is_multidimensional_array(type_: t.Type) -> bool:
+    """
+    Returns ``True`` if ``_type`` is list[list].
+    """
+    if t.get_origin(type_) is list:
+        args = t.get_args(type_)
+        if args and t.get_origin(args[0]) is list:
```

Comment:
I think this should be for checking a multidimensional list 
```python
if args[0] is list:
    return True
```
because `if args and t.get_origin(args[0]) is list:` always returns `False`

Reply:
Thanks - I need to test it.